src/main.ts

```diff
@@ -105,7 +105,11 @@ async function run(): Promise<void> {
 
       let body;
       if (useRawBody == 'true') {
-        body = output.trim();
+        // commentIntro needs to be present in body so that the bot edits
+        // existing comments and doesn't create new ones.
+        body = `<!-- ${commentIntro} -->
+        ${output.trim()}
+        `.trim();
```

Comment:
You could do this to avoid trimming again:

```suggestion
        ${output.trim()}`
```